package.json

```diff
@@ -19,11 +20,17 @@
     "react-dom": "16.8.1",
     "react-google-maps": "9.4.5",
     "react-jvectormap": "0.0.6",
+    "react-redux": "7.0.3",
     "react-router-dom": "4.3.1",
     "react-scripts": "2.1.5",
     "react-swipeable-views": "0.13.1",
     "react-table": "6.9.2",
-    "react-tagsinput": "3.19.0"
+    "react-tagsinput": "3.19.0",
```

Comment:
are we using this?

Reply:
I don't think so, i checked the project and this library is being used in a file css of the template that i bought.